rita_bin/src/exit.rs

```diff
@@ -90,6 +91,10 @@ fn main() {
         println!("Look the exit settings! {settings:?}");
         settings
     };
+    apply_babeld_settings_defaults(
```

Comment:
Do the exit settings files not also need to be migrated the way you are doing in client.rs?

Reply:
exits are at this point all hand configured, so not really as I'll update the exit installer, once we have appliance exits and an update procedure for those then yes we would need migration logic, so good catch.